core/templates/pages/profile-page/profile-page.component.ts

```diff
@@ -30,14 +30,14 @@ import { UserService } from 'services/user.service';
 import { ProfilePageBackendApiService } from './profile-page-backend-api.service';
 
 interface ViewedProfileUsername {
-  title: string;
-  value: string;
-  helpText: string;
+  title: string | null;
+  value: string | null;
+  helpText: string | null;
```

Comment:
Why we allow `null` here?

Reply:
I believe to reduce any chance of data of wrong data type being entered

Reply:
Okay, but I mean why these exact fields can have a `null` value? What does it mean specifically when `title` is `null`?

Reply:
Oh i get it resolving errors